lesson07/assignment/portfolioimages/App.vue

```diff
@@ -0,0 +1,216 @@
+<template>
+  <div id="app">
+<ul>
+<li v-for="link in links"> {{link}}</li>
+</ul>
+<div class=background><h1>Portfolio</h1>
+</div>
+
+<div class="gallery" v-for="image in gallery">
+  <img class="img" v-bind:src="image.picture" v-bind:alt="image.alt">
+  </div>
+
+<footer class="footer">
+  <ul>
+    <li><a href="./index.html">About</a><a href="./portfolioimages.html">Portfolio</a><a href="processaccordian.html">Process</a><a href="contactgrid.html">Book a Consultation</a></li>
+  </ul>
+</nav>
+<p><small>© 2022 Plant Portraits LLC | Seattle, WA</small></p>
+</footer>
+</div>
+</template>
+
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css"></script>
+
+<script>
+import app.vue from './vue.js/src/portfolioimages/app.vue'
+
+export default {
+  name: 'App',
+  data() {
+  return {
+  links: ['Home', 'Portfolio', 'Process', 'Book a Consultation'],
+  gallery:[
+  {picture: require('./images/plant1.jpeg'), alt:'Pink cherry blossom flower'};
+  {picture: require('./images/plant2.jpeg'), alt: 'Purple phlox'};
+  {picture: required('./images/plant3.jpeg'), alt: 'Hot pink peony blossom'};
+  {picture: required('./images/plant4.jpeg'), alt: 'Light purple foxglove flower'};
```

Comment:
Typos line 36 and 37, should be `require` not `required`. Again, this would have been easy to catch if the site had been manually tested.